hooks/commands/build.sh

```diff
@@ -15,23 +16,30 @@ image_file_name() {
 }
 
 push_image_to_docker_repository() {
-  local tag="$DOCKER_IMAGE_REPOSITORY:$(image_file_name)"
+  local tag="$BUILDKITE_PLUGIN_DOCKER_COMPOSE_BUILD_TAG"
 
-  plugin_prompt_and_must_run docker tag "$COMPOSE_SERVICE_DOCKER_IMAGE_NAME" "$tag"
   plugin_prompt_and_must_run docker push "$tag"
   plugin_prompt_and_must_run docker rmi "$tag"
```

Comment:
Well, if the user has provided an image name that is going to trump the generated tag name, it probably doesn't make sense to delete it does it? Otherwise it would have a cache miss the next time around.